src/app/shared/components/footer/footer.component.html

```diff
@@ -1,9 +1,12 @@
 <footer class="footer">
-  <p>Check out our social media accounts!</p>
-  <a target="blank" href="https://www.facebook.com/gdsc.upt">
-    <img src="/assets/facebook.png" alt="Facebook logo"/>
-  </a>
-  <a target="blank" href="https://dsc.community.dev/politehnica-university-of-timisoara/">
-    <img style="border-radius: 15px" src="/assets/logo_dsc.png" alt="Our logo"/>
-  </a>
+  <img class="logo" src="assets/logo.png" alt="logo">
+  <div class="icons">
+    <a target="blank" href="https://www.facebook.com/gdsc.upt">
+      <span class="icon icon-fb"></span>
+    </a>
+    <a target="blank" href="https://github.com/dsc-upt">
+      <span class="icon icon-github"></span>
+    </a>
+  </div>
+  <p>Made with <span style="color: red"> ❤ </span> by GDSC UPT Developers</p>
```

Comment:
Please use mat-icon instead of the heart and color the icon red.
Also place all the styling in the CSS file, even if it is only one property so things don't get confusing.
Make this text bigger on desktop